### PR TITLE
chore: add logs

### PR DESCRIPTION
**Pull request by jiacai2050**
# Which issue does this PR close?

Closes #

# Rationale for this change
 
When troubleshooting in local my host, I found those logs are useful.

# What changes are included in this PR?

# Are there any user-facing changes?

No

# How does this change test

This is what looks now
```
2023-02-06 16:07:24.514 INFO [server/src/http.rs:462] HTTP server starts listening on 127.0.0.1:5440
2023-02-06 16:07:24.515 INFO [server/src/server.rs:114] Server start, open local tables
2023-02-06 16:07:24.515 INFO [server/src/server.rs:128] Server start, create default schema if not exist
2023-02-06 16:07:24.516 INFO [server/src/server.rs:131] Server start, start services
2023-02-06 16:07:24.516 INFO [server/src/mysql/service.rs:54] MySQL server starts listening on 127.0.0.1:3307
2023-02-06 16:07:24.516 INFO [server/src/server.rs:138] Server start finished
2023-02-06 16:07:24.516 INFO [server/src/grpc/mod.rs:168] Grpc server starts listening on 127.0.0.1:8831
2023-02-06 16:07:24.517 INFO [server/src/grpc/mod.rs:177] Grpc server serves remote engine rpc service
```

### CODE REVIEW

File: server/src/mysql/service.rs [hunk 1]
```diff
@@ -51,14 +51,14 @@ impl<Q: QueryExecutor + 'static> MysqlService<Q> {
 
         let rt = self.runtimes.clone();
         self.tx = Some(tx);
+        info!("MySQL server starts listening on {}", self.socket_addr);
```
Comment (Rachelint) [inline server/src/mysql/service.rs:54]: Why move it to here?

File: server/src/http.rs [hunk 2]
```diff
@@ -459,6 +459,11 @@ impl<Q: QueryExecutor + 'static> Builder<Q> {
             enable_tenant_as_schema: self.enable_tenant_as_schema,
         };
 
+        info!(
+            "HTTP server starts listening on {}",
+            &self.config.endpoint.to_string()
+        );
```
Comment (Rachelint) [inline server/src/http.rs:465]: How about printing this after spawning?

File: components/message_queue/src/kafka/kafka_impl.rs [hunk 3]
```diff
@@ -211,6 +211,7 @@ impl MessageQueue for KafkaImpl {
             )
             .await;
 
+        info!("Create topic finish, name:{}", topic_name);
```
Comment (Rachelint) [inline components/message_queue/src/kafka/kafka_impl.rs:214]: I think the result should be printed in this log.

File: server/src/http.rs (hunk 2 above)
Comment (ShiKaiWi) [reply server/src/http.rs:465]: I guess you won't know which port will be used by the http service, so how about updating this log to:
```suggestion
        info!(
            "HTTP server tries to listen on {}",
            &self.config.endpoint.to_string()
        );
```

And add one more log after it succeeds in binding:
```
        info!(
            "HTTP server listens on {}",
            &self.config.endpoint.to_string()
        );
```

File: server/src/mysql/service.rs (hunk 1 above)
Comment (ShiKaiWi) [reply server/src/mysql/service.rs:54]: Maybe we should keep both the logs.

File: server/src/http.rs (hunk 2 above)
Comment (jiacai2050) [reply server/src/http.rs:465]: The log is verbose, I prefer to only add the first one.

Comment (jiacai2050) [reply server/src/http.rs]: I add panic logic when listen failed for grpc and mysql, http server already does this.

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Pull request merged**